src/components/App.jsx

```diff
@@ -1,70 +1,34 @@
-import React, { Component } from 'react';
+import React from 'react';
 import {
   BrowserRouter as Router,
   Route,
 } from 'react-router-dom';
 
-import { firebase } from '../firebase';
 import Navigation from './Navigation.jsx';
 import LandingPage from './Landing.jsx';
 import SignUpPage from './SignUp.jsx';
 import SignInPage from './SignIn.jsx';
 import PasswordForgetPage from './PasswordForget.jsx';
 import HomePage from './Home.jsx';
 import AccountPage from './Account.jsx';
-import * as routes from '../constants/routes';
-
+import withAuthentication from './withAuthentication.jsx';
 
-class App extends Component {
-  constructor(props) {
-    super(props);
+import * as routes from '../constants/routes';
 
-    this.state = {
-      authUser: null,
-    };
-  }
+const App = () => (
+  <Router>
+    <div>
+      <Navigation />
+      <hr />
+      <Route exact path={routes.LANDING} component={() => <LandingPage />} />
```

Comment:
Why don't you define the routes elsewhere and mount it dynamically iterating over the list?